core-web/libs/edit-content/src/lib/services/dot-edit-content.service.ts

```diff
@@ -44,6 +44,19 @@ export class DotEditContentService {
         );
     }
 
+    /**
+     * Retrieves tags based on the provided name.
+     * @param name - The name of the tags to retrieve.
+     * @returns An Observable that emits an array of tag labels.
+     */
+    getTags(name: string) {
+        const params = new HttpParams().set('name', name);
+
+        return this.http.get('/api/v2/tags', { params }).pipe(
+            pluck('entity'),
+            map((res) => Object.values(res).map((obj) => obj.label))
```

Comment:
What happens for an empty/nullish object? In the empty starter `res` could be `null/undefined/{}`

Reply:
In the empty starter the res structure is different? Because if the res is empty (dont have tag suggestion for the search), res = {}, so the code isnt break

But with empty starter, the response can be different?

Reply:
I'm not really sure what this returns for no tags, I tried to delete everything in my environment and hit the endpoint, but I think the system has some tags hardcoded in the starter because It still gave me some tags

Reply:
<img width="232" alt="Screenshot 2023-12-11 at 15 02 30" src="https://github.com/dotCMS/core/assets/144152756/830e346e-4923-4d06-8a55-4ab51975ac7a">
This is a response from a random non-existen tag
I doubt it is different in the empty starter, but idk
cc: @fmontes  opinion?

Reply:
In that case is safe to leave it as it is. `Object.values` will return an empty array

Reply:
Empty will be the same.